docs/source/rules.rst

```diff
@@ -62,18 +63,52 @@ contain such values e.g.
             type: "eq"
             pattern : "customvalue"
 
+Managing Rules
+--------------
+
+To deploy a rule, use CLI ``st2 rule create ${PATH_TO_RULE}`` command, for example:
+
+.. code-block:: bash
+
+    st2 rule create /usr/share/doc/st2/examples/rules/sample_rule_with_webhook.yaml
+
+If the rule with the same name already exists, the above command will return an error:
+
+.. code-block:: bash
+    
+    ERROR: 409 Client Error: Conflict
+    MESSAGE: Tried to save duplicate unique keys (E11000 duplicate key error index: st2.rule_d_b.$name_1  dup key: { : "examples.webhook_file" })
+        
+To update the rule, edit the rule definition file and run ``st2 rule update`` command, as in the following example:
```

Comment:
I believe when doing update via file, file also needs to contain object id (please correct me if I'm wrong).

If that's indeed the case, we should point this out here.